api/app/Repositories/Event/EventRepository.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories\Event;
+
+use App\Entity\Event;
+use App\Repositories\BaseRepository;
+
+final class EventRepository extends BaseRepository implements EventRepositoryInterface
+{
+    public function save(Event $event)
+    {
+        $event->save();
```

Comment:
To be consistent with other repositories, the save/update method must return the saved entity.